Sources/Core/Services/NowPlayingService.swift

```diff
@@ -44,11 +44,14 @@ final class ModernAVPlayerNowPlayingService: NowPlaying {
     }
     private var task: URLSessionTask?
 
-    func update(media: PlayerMedia, duration: Double?) {
-        infos = parseInfos(media: media, duration: duration)
+    func update(metadata: PlayerMetadata?, duration: Double?, isLive: Bool) {
+        infos = parseInfos(metadata: metadata, duration: duration)
+        if #available(iOS 10, *) {
```

Comment:
why move this execution here ? not a good place in parseInfos() method ?

Reply:
ParseInfo now takes metadata input, as he should. 
isLive is a property of the stream, not the metadata.
Wdyt ?

Reply:
parseInfos build a new NowPlayingInfo dictionnary. we may first rename the method.
then isLive is a property of the stream, right but this attribute is used as an attibute of the dictionnary

Wdyt ?